src/main/java/pedido/Cardapio.java

```diff
@@ -5,38 +5,63 @@
 import java.util.TreeMap;
 
 public class Cardapio {
-    private TreeMap<Ingrediente,Double> precos;
+
+    private final TreeMap<Ingrediente,Double> precos;
 
     public Cardapio(){
         this.precos= new TreeMap<>();
     }
 
-    public TreeMap<Ingrediente, Double> getPrecos(){
-        return this.precos;
-    }
-
     public void adicionarIngrediente(Ingrediente ingrediente,Double preco){
-        //TODO
+        if(preco <= 0){
+            throw new IllegalArgumentException("Preco invalido.");
```

Comment:
Como você precisa verificar se o preço é válido em outros momentos também, da para criar um método para validar o preço e se caso não for válido já lançar a exceção, porque daí você evita repetir o mesmo código todas as vezes que precisa validar o preço.